docs/js-core-1/preparation.md

```diff
@@ -0,0 +1,96 @@
+3. Ensure that you have succesfully installed Homebrew by typing `brew -v` in your terminal - you should get an output that says something like "`Homebrew <version number>`"
+4. Install NodeJS and npm by typing `brew install node`
+5. Check that you have successfully installed NodeJS by ensuring that the following command returns "v13.8.0": `node -v`
+6. Check that you have successfully installed npm by ensuring that the following command returns "6.13.6": `npm -v`
+
+## On Ubuntu
+
+1. Type ctrl + alt + T to open up the terminal
+2. Install NodeJS and npm by copying and pasting the following into the terminal: `sudo apt-get install curl && curl -sL https://deb.nodesource.com/setup_13.x | sudo -E bash - && sudo apt-get install nodejs`
+3. Check that you have successfully installed NodeJS by ensuring that the following command returns "v13.8.0": `node -v`
+4. Check that you have successfully installed npm by ensuring that the following command returns "6.13.6": `npm -v`
+
+## On Windows
+
+1. Open Ubuntu WSL (click Windows button then type "Ubuntu")
+2. Follow the instructions from the "On Ubuntu" section starting from step 2
+
+## 3) KhanAcademy Review (4 hours)
+
+We are going to review the essentials of the Khan Academy course that we learned during the Fundamentals course.
+
+### 3.1) Reading
+
+Please read the following materials
+
+- Variables (Review): https://www.khanacademy.org/computing/computer-programming/programming/variables/a/review-variables
+- Functions (Review): https://www.khanacademy.org/computing/computer-programming/programming/functions/a/review-functions
+- Logic and if Statements (Review): https://www.khanacademy.org/computing/computer-programming/programming/logic-if-statements/a/review-logic-and-if-statements
+- Looping (Review): https://www.khanacademy.org/computing/computer-programming/programming/looping/a/review-looping
+- Arrays (Review): https://www.khanacademy.org/computing/computer-programming/programming/arrays/a/review-arrays
+
+If you think you need more time to re-assimilate these concepts you can go over the video tutorials again.
+
+### 3.2) Project
+
+Create a spin-off of the following project
+
+https://www.khanacademy.org/computer-programming/bubbles/6227974791118848
+
+You'll see there's a fish and a bubble.
+
+- Add more bubbles of different sizes in random positions.
+- BONUS: Animate the bubbles.
+- BONUS: Animate the fish.
+- BONUS: Make the fish turn back after hitting the edge of the tank.
+
+Try to use in your solution the constructions you just reviewed: variables, functions, if statements, loops or arrays. You can adapt and reuse any code from your previous projects.
+
+## 4) Visual Code
+
+To make your programming life a lot easier, you should install Prettier on VS Code (which will automatically make your code look more "neat" and readable) and enable autosaving on VS Code (so that you don't forget to save your file before you run your script).
```

Comment:
It seems weird to be telling people to install Prettier on VS Code, and _then_ be telling them to install VSCode?